lib/web_ui/lib/semantics.dart

```diff
@@ -282,6 +282,7 @@ class SemanticsUpdateBuilder {
     required Int32List childrenInTraversalOrder,
     required Int32List childrenInHitTestOrder,
     required Int32List additionalActions,
+    required int headingLevel,
```

Comment:
This must be consistent with the mobile version (the mobile version is `int? headingLevel`, not `required int`).

Reply:
Oops!, changed, thank you.